### PR TITLE
Setup/tailwind

### LINKED ISSUE #5: Setup Tailwind

### PR DESCRIPTION
**Pull request by campbellkearns**
closes #5

### CODE REVIEW

File: src/styles/globals.css [hunk 1]
```diff
@@ -1,16 +1,3 @@
...
-  color: inherit;
-  text-decoration: none;
-}
-
-* {
-  box-sizing: border-box;
-}
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (saihaj) [inline src/styles/globals.css:3]: new line : )

File: tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const defaultTheme = require('tailwindcss/defaultTheme')
+
+module.exports = {
+  purge: [
+    './src/**/*.jsx',
```
Comment (saihaj) [inline tailwind.config.js:5]:
```suggestion
    './src/**/*.{js,jsx,tsx}',
```

**Review by saihaj — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 2 above)
Comment (campbellkearns) [reply tailwind.config.js]: nice

**Review by saihaj — APPROVED** [review]
Awesome! 🎉

**Pull request merged**